server/src/api/mutation/upsertUserPersonalization.ts

```diff
@@ -0,0 +1,26 @@
+import { badRequest, forbidden } from '@hapi/boom';
+
+import { upsertUserPersonalization } from '../../functions';
+import { MutationResolvers } from '../../graphql';
+import { userPersonalizationSchema } from '../schema';
+
+type Resolver = MutationResolvers['upsertUserPersonalization'];
+
+export const resolver: Resolver = async (
+  _,
+  { userPersonalization },
+  { req },
+) => {
+  if (req.userId !== userPersonalization.user_id) {
```

Comment:
```
  if (req.userId !== userPersonalization.user_id) {
    throw forbidden();
```
Remove the authenication check here since user_personalization no longer has user_id
This is because the user will already be authenticated once he/she reaches this page.